nanoid v5 - async API removed

### Environment

-

### Reproduction

-

### Describe the bug

In the latest version of nanoid, they removed `nanoid/async` module
https://github.com/ai/nanoid/blob/main/CHANGELOG.md#50
Hence Pinia ORM (master branch as of now) build failing:
```
src/packages/nanoid/casts/AsyncCast.ts(1,24): error TS2307: Cannot find module 'nanoid/async' or its corresponding type declarations.
```
I think we should remove it too.

### Additional context

_No response_

### Logs

_No response_

--- a/packages/axios/package.json
+++ b/packages/axios/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@pinia-orm/axios",
-  "version": "1.10.1",
+  "version": "2.0.0-alpha.1",
   "description": "Axios plugin for pinia-orm",
   "bugs": {
     "url": "https://github.com/CodeDredd/pinia-orm/issues"

--- a/packages/normalizr/package.json
+++ b/packages/normalizr/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@pinia-orm/normalizr",
-  "version": "1.10.1",
+  "version": "2.0.0-alpha.1",
   "description": "Normalizes JSON according to schema for Redux and Flux applications",
   "bugs": {
     "url": "https://github.com/CodeDredd/pinia-orm/issues"

--- a/packages/nuxt/package.json
+++ b/packages/nuxt/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@pinia-orm/nuxt",
   "description": "Nuxt module for pinia-orm",
-  "version": "1.10.1",
+  "version": "2.0.0-alpha.1",
   "license": "MIT",
   "author": {
     "name": "Gregor Becker",

--- a/packages/pinia-orm/package.json
+++ b/packages/pinia-orm/package.json
@@ -1,6 +1,6 @@
 {
   "name": "pinia-orm",
-  "version": "1.10.1",
+  "version": "2.0.0-alpha.1",
   "description": "The Pinia plugin to enable Object-Relational Mapping access to the Pinia Store.",
   "keywords": [
     "vue",

